invenio_i18n/__init__.py

```diff
@@ -305,8 +302,10 @@
 
     security.safe_str_cmp = hmac.compare_digest
 
+from flask_babel import Babel, gettext, lazy_gettext
```

Comment:
we could, but in the comments of this patch it is stated that we should do this if `flask-security-fork` has `flask-login>=v0.6.0` which is not yet the case. It will be installed the newest because there is no upper limit, but it is not guaranteed that the necessary version will be installed. Maybe we should do that also, so we can remove not only the monkey patch in this package but also in others.